keyboards/westfoxtrot/cyclops/cyclops.h

```diff
@@ -35,9 +35,9 @@
 ) { \
 	{ K00,  K01,   K02,  K03,   K04,   K05,   K06,  K07,   K08,   K09,   K0A,  K0B,  K0C,  K0D,  K0E }, \
 	{ K10,  KC_NO, K12,  K13,   K14,   K15,   K16,  K17,   K18,   K19,   K1A,  K1B,  K1C,  K1D,  K1E }, \
-	{ K20,  KC_NO, K22,  K23,   K24,   K25,   K26,  K27,   K28,   K29,   K2A,  K2B,  K2C,  K2D,  K2E }, \
+	{ K20,  KC_NO, K22,  K23,   K24,   K25,   K26,  K27,   K28,   K29,   K2A,  K2B,  K2C,  K2D,  K4E }, \
```

Comment:
You are editing the electrical matrix here. That position that you changed can ONLY either be K2E or KC_NO.